tools/cxbe/Xbe.cpp

```diff
@@ -686,6 +697,36 @@ Xbe::Xbe(class Exe *x_Exe, const char *x_szTitle, bool x_bRetail)
             }
 
             hwc = hwc_secn;
+            szBuffer = m_HeaderEx + hwc - (m_Header.dwBaseAddr + sizeof(m_Header));
+        }
+
+        // Write library versions
+        {
```

Comment:
There should pobably be a comment that these are fake libraries.

Reply:
Would "Write (placeholder) library versions" work?